shared/chat/inbox/row/small-team/swipe-conv-actions/index.native.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react'
+import * as Kb from '../../../../../common-adapters/mobile.native'
+import * as Styles from '../../../../../styles'
+import {Props} from '.'
+
+let curSwipeRef: React.RefObject<Kb.Swipeable> | null = null
```

Comment:
i'd change this to 
```
let curSwipeRef = React.createRef<Kb.Swipeable>()
```

then when you want to null it just `curSwipRef.current = null`

Reply:
then `curSwipRef` itself is never null